src/dynamic/component-outlet-injector.directive.spec.ts

```diff
@@ -10,10 +10,12 @@ import {
 import { ComponentOutletInjectorDirective } from './component-outlet-injector.directive';
 
 @Component({
-  template: `<ng-container *ngComponentOutlet="comp"></ng-container>`,
+  template: `
+    <ng-container *ngComponentOutlet="comp"></ng-container>
+  `,
 })
 class TestComponent extends TestComponentBase {
-  @ViewChild(ComponentOutletInjectorDirective)
+  @ViewChild(ComponentOutletInjectorDirective, { static: false })
```

Comment:
This extra parameter is added in ng8. False checks after `ngAfterViewInit` and true checks on `ngOnInit`, the default will be set to false (as that's when, most of the time, you'll wanna check).